doc/tech/usecases.md

```diff
@@ -9,5 +9,36 @@
 ## 5.1 Core
 
 
-<a name="5.1"></a>
-## 5.1 Edge
+<a name="5.2"></a>
+## 5.2 Edge
+Principles:
+
+This section introduces principles that should be followed during the definition and development of the Edge scope for the CNTT Reference Model, Reference Architectures, Reference Implementations and Reference Conformance test suites.
+
+A main principle is that we shall not define a new branch of CNTT and refrain from inventing new things where other organizations already have solutions or ongoing suitable standardization. By that follows that the Edge Principles, as far as possible, shall follow and amend the existing [Reference Model Principles](../ref_model/chapters/chapter01.md#13-principles), the [Reference Architecture Principles](../ref_arch#principles) and the Network Principles that are being developed in parallel. 
+
+There are however a couple of new things that Edge deployments bring into the picture compared to the larger or regional Cloud Infrastructure deployments. These differences will generate a small set of additional principles. For now, there are only a few high level mixed differences and principles listed to generate suitable debate and feedback, but it will be filled out with more clarifications over time and during the development of the prioritized use cases.
```

Comment:
what does this mean, "For now, there are only a few high level mixed differences .." -- what are "mixed differences"?

Reply:
i don't know :):):):):)
it is completely not understandable :):)

let me try to rephrase it